.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,130 @@
+name: Run sytest
+on:
+  push:
+    branches: ["develop", "release-*"]
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  synapse:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - sytest-tag: bionic
```

Comment:
I don't suppose this is easy, but is it possible to additionally include the Python version in each job name?

![image](https://user-images.githubusercontent.com/1342360/129345844-bc366f8f-8056-482a-8a7b-7e65073b5078.png)

Reply:
Wasn't too bad: see https://github.com/DMRobertson/sytest/actions/runs/1127951256

![Screenshot from 2021-08-13 15-16-40](https://user-images.githubusercontent.com/8614563/129370959-6515de8d-d3c2-4a0e-b4e2-c56623c458a8.png)